### PR TITLE
Adding new DMCA Policy page

### PR DESCRIPTION
**Pull request by clone1018**
screencapture-glimesh-dev-about-dmca-1610244657724

### CODE REVIEW

**Review by Heiwa1580 — CHANGES_REQUESTED** [review]
Sentence structure in some areas should be improved. Also some grammer errors.

Also if we are looking at this sort of format it may be nice to beef up some of the explanations just a little to make it more in line. For example it mentions nothing about the timelines of our takedowns or the timeline of the counter notices or anything about a repeat infringer policy.

**Comment by clone1018** [comment]
You should be able to make revisions directly in the Files Changed tabs, it'll be easier on a desktop though :)

**Comment by Heiwa1580** [comment]
> You should be able to make revisions directly in the Files Changed tabs, it'll be easier on a desktop though :)

Haha yeah when I find the time I will for sure.

File: lib/glimesh_web/templates/about/dmca.html.eex [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+                        </li>
+                        <li>
+                            <%= gettext("Glimesh will be clear as to how the rules work. There will be no guessing as to how to counter notice or how the process works. Also Glimesh will require that takedown notices include very specifically where the copyright content is so that you are not guessing what you may have a strike for.") %>
+                        </li>
+                        <li>
+                            <%= gettext("Glimesh aims to be fair as possible. While we cannot condone improper use of copyright material we know that not all DMCA claims are fair. We will be open to help in whatever ways possible to ensure you have the best opportunity to correct DMCA claims. While we are not lawyers and cannot offer legal advice we will do our best to help you get in the right direction for correcting any claims in a fair manner.") %>
+                        </li>
+                        <li>
+                            <%= gettext("Glimesh is also working towards developing ideas where we can either satisfy licensing requirements.") %>
```
Comment (Heiwa1580) [inline lib/glimesh_web/templates/about/dmca.html.eex:198]:
```suggestion
                            <%= gettext("Lastly Glimesh is working to partner with music providers or independant artists who will provide properly licensed music for you to use.") %>
```

File: lib/glimesh_web/templates/about/dmca.html.eex [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+
+
+            <img src="<%= Routes.static_path(@conn, "/images/about/dmca/1.png") %>" alt="Divider" class="img-fluid" />
+
+            <h3><%= gettext("What are Copyrights?") %></h3>
+            <div class="row justify-content-md-center mt-4">
+                <div class="col-md-10">
+
+                    <p>
+                        <%= gettext("Copyrights are protections given to creative works and their owners. You can simply think of copyrights as the right to copy a creative work. As the owner of a creative work you have the right to copy that work in any way you'd like and copyright laws like the DMCA protect you from anyone else copying that work without your permission. Copyrights are actually given to you as the owner the moment you create a work. You can get official copyrights to register things by law but by default the law gives you some basic copyrights the moment you create an original work.") %>
```
Comment (Heiwa1580) [inline lib/glimesh_web/templates/about/dmca.html.eex:34]:
```suggestion
                        <%= gettext("Copyrights are protections given to creative works and their owners. You can simply think of copyrights as the right to copy a creative work. As the owner of a creative work you have the right to copy that work in any way you'd like and copyright laws like the DMCA protect you from anyone else copying that work without your permission. Copyrights are actually given to you as the owner the moment you create a work. By default the law gives you some basic copyrights the moment you create an original work registering your work online just makes it easier to protect but creative works are instantly protected by Copyright laws.") %>
```

File: lib/glimesh_web/templates/about/dmca.html.eex [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+            <div class="row justify-content-md-center mt-4">
+                <div class="col-md-10">
+
+                    <p>
+                        <%= gettext("Glimesh aims to be fully compliant to the DMCA and because of that a lot of our processes will look similar to others. Here are the highlights though of how Glimesh will be different.") %>
+                    </p>
+
+                    <ul>
+                        <li>
+                            <%= gettext("Glimesh will be fully transparent about DMCA Takedown requests. The whole community will see the requests and will be able to respond to them. So false requests will have little effectiveness and the community as a whole will be able to combat them together. ") %>
```
Comment (Heiwa1580) [inline lib/glimesh_web/templates/about/dmca.html.eex:189]:
```suggestion
                            <%= gettext("Glimesh will be fully transparent about DMCA Takedown requests. The whole community will see the requests and will be able to respond to them. So false requests will have little effectiveness and the community as a whole will be able to combat them together. ") %>
```

**Pull request merged**